### ISSUE TITLE
tabular data/ noisy instances 

### ISSUE THREAD

**Issue by nazaretl** [502463708/Confident_Learning_for_Noisy-labeled_Medical_Image_Segmentation#4]
Hi,
thanks for sharing your implementation. I have two questions about it:

1.	Does it also work on tabular data?
2.	Is it possible to identify the noisy instances (return the noisy IDs or the clean set)?

Thanks!

**Comment by 502463708** [comment]
1. what do you mean by mentioning tabular data
2. yes

**Comment by nazaretl** [comment]
non-image data, e.g. Iris dataset, where convolutions are not possible/ meaningfull

**Comment by 502463708** [comment]
probably yes as long as you got a probabilistic model, because our method works on the prediction results

**Comment by nazaretl** [comment]
I see, thank you. But for that the GitHub implementation has to be rewritten right? It seems that it is tailored to 3 datasets (there is no dataset-independent implementation so far).

Could you send me your paper please? I don't have access to it via the link in the repository.

**Comment by 502463708** [comment]
It is now available in Google Scholar.
